serendipity_event_markdown/serendipity_event_markdown.php

```diff
@@ -237,6 +251,49 @@ function _markdown_markup($text) {
         return Markdown($text);
     }
     */
+
+    function addToCSS(&$eventData) {
+        $eventData .= '
+/* Footnotes (generated by serendipity_event_markdown) */
+
+a.footnote-ref:after {
```

Comment:
Does is *have* to be a combined selector for specifity or would `.footnote-ref:after` suffice?